package.json

```diff
@@ -43,7 +43,7 @@
     "@podium/schemas": "5.0.6",
     "@podium/utils": "5.2.0",
     "abslog": "2.4.4",
-    "path-to-regexp": "6.2.2"
+    "path-to-regexp": "8.1.0"
```

Comment:
I see that since this PR was opened, the maintainer has backported the fix to 6.x as well:
https://github.com/pillarjs/path-to-regexp/releases/tag/v6.3.0

If we feel this is too risky for a patch release we could adopt that one instead.